java-squid/src/main/java/org/sonar/java/model/JavaTree.java

```diff
@@ -605,5 +629,25 @@ public void accept(TreeVisitor visitor) {
     public Iterator<Tree> childrenIterator() {
       return Iterators.<Tree>singletonIterator(type);
     }
+
+    @Override
+    public List<AnnotationTree> annotations() {
+      return (List<AnnotationTree>) annotations;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unnecessary cast to "List". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1905)